rs/rosetta-api/icrc1/index-ng/tests/retrieve_blocks_from_ledger_interval.rs

```diff
@@ -237,8 +237,8 @@ fn should_consume_expected_amount_of_cycles() {
             initial_interval: Some(DEFAULT_MAX_WAIT_TIME_IN_SECS),
             upgrade_interval: Some(DEFAULT_MAX_WAIT_TIME_IN_SECS),
             assert_cost: |initial, upgrade| {
-                const EXPECTED_LEDGER_CYCLES_CONSUMPTION: i128 = 57_473_930;
-                const EXPECTED_INDEX_CYCLES_CONSUMPTION: i128 = 207_959_396;
+                const EXPECTED_LEDGER_CYCLES_CONSUMPTION: i128 = 124_445_767;
```

Comment:
Those tests are using a setup with 28 nodes with the default of 13. After turning on the scaling factor for `state_machine_tests` framework corresponding costs go up ~2.154 times.

```
28 / 13 = 2.154
124_445_767 / 57_473_930 = 2.165
449_388_554 / 207_959_396 = 2.161
```

Reply:
Thanks for updating the test and for the explanation, @maksymar!